rpc/core/types/responses.go

```diff
@@ -101,6 +101,14 @@ type SyncInfo struct {
 
 	TotalSyncedTime time.Duration `json:"total_synced_time"`
 	RemainingTime   time.Duration `json:"remaining_time"`
+
+	TotalSnapshots     int64         `json:"statesync_total_snapshots"`
+	ChunkProcess       time.Duration `json:"statesync_chunk_process"`
+	SnapshotHeight     int64         `json:"statesync_snapshot_height"`
+	SnapshotChunk      int64         `json:"statesync_snapshot_chunk"`
+	SnapshotChunkTotal int64         `json:"statesync_snapshot_chunk_total"`
+	BackFill           int64         `json:"statesync_backfill"`
+	BackFillTotal      int64         `json:"statesync_backfill_total"`
```

Comment:
Why are these prefixed with `statesync_`?

Reply:
I was thinking these data belong to the statesync category, so I added the prefix. I can remove it if it's not appropriateㄡ